pages/implementation/reference/javascript.mdx

```diff
@@ -0,0 +1,190 @@
+---
+layout: 'guide'
+title: 'JavaScript'
+---
+
+import Link from '../../../components/Link';
+
+### Platforms
+
+Avo can generate JavaScript code targeted at the following platforms
+
+- Web
+- React Native
+- Node.js
+
+### Quickstart
+
+Avo functions usage consists of 3 steps.
```

Comment:
For future: Here we can add link to the "Type safe code & unit tests" docs section when that's ready, something like:
> Learn more about how Avo Functions can help decrease time spent on implementing analytics and increase data reliability in this guide.

Reply:
Moved the future plans to TODO comments